src/pages/hegel/reference/the-mechanical-process/the-formal-mechanical-process-section-1.mdx

```diff
@@ -0,0 +1,243 @@
+distinction between a scented molecule and a non-scented molecule does not
+obtain within a mechanistic framework. This mode of thinking about how the
+logical determinations are reflected in everyday occurrences suggests that the
+same object can be thought of within a mechanistic framework, as well as other
+frameworks. Take the human body, one could conceive of it purely in mechanistic
+terms and theories of its processes in terms of cause and effect, or one could
+conceive of it in organicist terms and conceive of its processes differently.
+Now, what is crucial about this example is that it works because mechanistic
+thinking is more reductive than organicist thinking – the reverse would not be
+possible. One could not conceive of a rock within an organicist framework. Thus,
+the possibility of an object to be conceived within a plurality of frameworks
+requires that the object is open to reductionism.
+
+Now, a further thought. This does not mean that it is equally correct to
+conceive of the same object through the prism of multiple frameworks. The human
+body is an organic thing and so to conceive of it as a `mechanical object` is to
+misconstrue it. It _can_ be done but you are failing to fully grasp what it is a
+for a human body to be what it is. Let us hold onto this reflection and turn
+back to the example of the diffusion of scent. Since we are within the realm of
+Mechanism, the simplest way that we have to conceive of the scented and the
+non-scented molecules is as `mechanical objects`[^2]: Of course, one could
+conceive of them as a `syllogism` or a `judgement`, but then they would cease to
+be conceived of as objects. Instead, they would be conceived of as concepts. It
+is plausible that one would have a more accurate conception of them through a
+chemical framework, as found in Chemism, which actually takes into account their
+different determinateness. But for now, they are `mechanical objects` whose
+determinateness is identical.
+
+### The Ideality of Communication
+
+Returning to the text and progressing with it, Hegel writes that: “But even in
+communication between material objects, their determinateness spreads, so to
+speak, in a similarly ideal manner” (Hegel 1991, 716). When talking about the
+diffusion of scent amongst molecules one might get the false impression that
+`communication` is the kind of process that only works with ideal, non-material
+objects. Now, It looks as if in Hegel’s time scent was thought to be transmitted
+in a non-material way. Obviously, this is no longer the case, but Hegel’s point
+is clear: `communication` is an ideal process that occurs between material
+objects. In other words, it is not just material processes that occur between
+material objects. Here, Hegel is clearly going against a materialist/mechanistic
+worldview whereby the only processes are _material_ processes. Perhaps Hegel is
+thinking of forces and, perhaps more specifically, the force of gravity as an
```

Comment:
Is there any indication of this? Force specifically belongs to the Doctrine of Essence. Gravity also becomes exemplified later in Mechanism if I recall correctly, so Hegel can't be thinking of gravity here (yet).

Reply:
Interesting points. My thoughts are as follows: 
(1) I'm talking about force as it is used in science, not Logic. Now, it may be that Hegel's account of force in the DOE is his fundamental account of force as it is used in science. But it would be strange if it was the exhaustive account of force, because it does look as if Hegel is giving some account of force in Mechanism. Now that raises some interesting questions. (1) Is Hegel's account of force in DOE his account of forces in general? (2) If not, what is it about? If yes, what does that mean for his apparent account of forces in Mechanism? 

I'm not sure what the answers to the above are. From my perspective, I would say that Hegel is giving an account of a specific force or forces in Mechanism. The reason why I think communication could be a good candidate for gravity is because of the all pervasive nature of gravity. Incidentally, I don't think his account in Absolute Mechanism is meant to evoke gravity, I think it evokes the operation of a system. The underlying force of that system may be gravity, but gravity can work on objects in a non-systematic way (when a rock falls to the Earth). Whereas in Absolute Mechanism it is working on objects in a thoroughly systematic way (when the planets rotate around the Sun).

Reply:
Good stuff, I'll save this for a discussion thread in case we want to pick it up again :) Might be good to revisit when you get to absolute mechanism.

Reply:
I'll add this into a discussion. Again, thanks a lot for the question, it prompted me to think more deeply about this.

Reply:
You already did it. What a lad.